docs/ops/deployment/yarn_setup.md

```diff
@@ -132,7 +132,7 @@ Stop the YARN session by stopping the unix process (using CTRL+C) or by entering
 
 Flink on YARN will only start all requested containers if enough resources are available on the cluster. Most YARN schedulers account for the requested memory of the containers,
 some account also for the number of vcores. By default, the number of vcores is equal to the processing slots (`-s`) argument. The `yarn.containers.vcores` allows overwriting the
```

Comment:
why not replace "`yarn.containers.vcores`" with the link?

Reply:
The way it is right now it implies there is more information on the CPU scheduling in the description of the parameter.

Reply:
I agree with @zentol - a link here (in addition) would be better to simply click your way through the docs